examples/experiment.rs

```diff
@@ -0,0 +1,29 @@
+use anyhow::Result;
+use experiments::Experiments;
+fn main() -> Result<()> {
+    viaduct_reqwest::use_reqwest_backend();
+    let exp = Experiments::new(
+        "https://kinto.dev.mozaws.net/v1/",
```

Comment:
let's move the setting of server down into the crate. It can be hardcoded for now, until we get support for something like `ServerConfig`. per comment in https://github.com/a-s-dev/android-components/pull/1#discussion_r458785483